cmd/gen-docs/main.go

```diff
@@ -7,23 +7,60 @@ import (
 
 	"github.com/cli/cli/command"
 	"github.com/spf13/cobra/doc"
+	"github.com/spf13/pflag"
 )
 
 func main() {
-	if len(os.Args) < 2 {
-		fatal("Usage: gen-docs <destination-dir>")
+
+	var flagError pflag.ErrorHandling
+	docCmd := pflag.NewFlagSet("", flagError)
+	var manPage = docCmd.BoolP("man-page", "", false, "Generate manual pages")
```

Comment:
is there a reason to not define all of these like: `manPage := docCmd.BoolP()`

Reply:
No idea, this is my first time touching go, and I didn't even notice I copied two declaration formats.
I'll go clean that up.